components/identity.oauth.uma.permission.endpoint/src/gen/java/org/wso2/carbon/identity/oauth/uma/endpoint/ApiException.java

```diff
@@ -0,0 +1,9 @@
+package org.wso2.carbon.identity.oauth.uma.endpoint;
+//comment
```

Comment:
?

Reply:
this was auto generated

Reply:
This is auto generated. Does it need to be removed?